gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Tue Aug 25 10:55:16 JST 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.6-all.zip
+distributionSha256Sum=11657af6356b7587bfb37287b5992e94a9686d5c8a0a1b60b87b9928a2decde5
```

Comment:
@cristiangreco 
Is this line needed ?

Reply:
@hannesa2 This property is added by the action to harden the Gradle configuration. In fact, when this line is present, an additional check will be performed the first time you download the new Gradle version locally: the value here is the expected sha256 checksum of the Gradle binary that the Wrapper will download from Gradle's server -- in case of a computation mismatch (e.g. the remote file has been tampered with) Gradle will output a warning message on your terminal.

You can find more info on Gradle's website [here](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:verification). 

The `distributionSha256Sum` property is added for your own security, but if you really want to disable this behaviour in the UGW action use the [`set-distribution-checksum`](https://github.com/gradle-update/update-gradle-wrapper-action#set-distribution-checksum) input (but I'd not recommend disabling it, for the reasons above).